packages/fondue/src/components/LoadingBar/LoadingBar.tsx

```diff
@@ -44,6 +44,9 @@ export const loadingBarStateClasses: Record<LoadingBarState, { outer: string; in
     },
 };
 
+/**
+ * @deprecated Please use updated LoadingBar component from @frontify/fondue/components
```

Comment:
Same for the Divider and Button please
```suggestion
 * @deprecated Please use updated LoadingBar component from `@frontify/fondue/components`.
```